widgetssdk/src/main/java/com/glia/widgets/view/OperatorStatusView.java

```diff
@@ -115,22 +116,40 @@ public void showPlaceHolderWithIconPaddingOnConnect() {
 
     public void showProfileImageOnConnect(String profileImgUrl) {
         updateProfilePictureViewSize(operatorImageLargeSize);
-        Picasso.get().load(profileImgUrl).into(profilePictureView);
-        updatePlaceholderView(
-                operatorImageLargeSize,
-                NO_PADDING,
-                GONE
-        );
+        Picasso.get().load(profileImgUrl).into(profilePictureView, new Callback() {
+            @Override
+            public void onSuccess() {
+                updatePlaceholderView(
+                        operatorImageLargeSize,
+                        NO_PADDING,
+                        GONE
+                );
+            }
+
+            @Override
+            public void onError(Exception e) {
+
+            }
+        });
     }
 
     public void showProfileImage(String profileImgUrl) {
         updateProfilePictureViewSize(operatorImageSize);
-        Picasso.get().load(profileImgUrl).into(profilePictureView);
-        updatePlaceholderView(
-                operatorImageSize,
-                NO_PADDING,
-                GONE
-        );
+        Picasso.get().load(profileImgUrl).into(profilePictureView, new Callback() {
```

Comment:
According to [Picasso docs](https://square.github.io/picasso/2.x/picasso/com/squareup/picasso/RequestCreator.html#into-android.widget.ImageView-com.squareup.picasso.Callback-):
`Note: The Callback param is a strong reference and will prevent your Activity or Fragment from being garbage collected. If you use this method, it is strongly recommended you invoke an adjacent Picasso.cancelRequest(android.widget.ImageView) call to prevent temporary leaking.
`

So, it looks like, with the usage of callback, there's a need to make `Picasso.cancelRequest` at some point, isn't it?

Reply:
cancels request on detached